packages/appkit/tests/appkit.test.ts

```diff
@@ -1037,7 +1037,7 @@ describe('Base', () => {
 
       await (appKit as any).syncExistingConnection()
 
-      expect(ChainController.showUnsupportedChainUI).not.toHaveBeenCalled()
+      expect(ChainController.showUnsupportedChainUI).toHaveBeenCalled()
```

Comment:
how come?

Reply:
test is saying it should NOT call it

Reply:
@tomiir It's been added to test the workflow check if it's going to block E2E if unit tests fails. For testing purposes